### ISSUE TITLE
purchaseState or state?

### ISSUE THREAD

**Issue by daniefvh** [AlexDisler/cordova-plugin-inapppurchase#258]
This may only be in documentation:

When calling restorePurchases, in the payload the state field is called purchaseState but the documentation indicates it is called state.

**Comment by netse-ai** [comment]
Does restorePurchases() work for you??? I only get "Error retrieving purchase details". This is even with active, REAL subscriptions.

**Comment by daniefvh** [comment]
It works but the state doesn't seem to reflect the purchase has been refunded on android at least.

I noticed there is a state on the main payload and a purchaseState in the receipt but both seems to be wrong.

@tear727 did you update your manifest.json file with the play_store_key? That tripped me up initially

**Comment by netse-ai** [comment]
@daniefvh Yes, the key was actually the issue. Google changed my key for some reason. I hadn't used it in a few months so maybe they update them when they aren't used.
